HW4/src/getClosestToZero.js

```diff
@@ -0,0 +1,3 @@
+const closest = (array) => array.sort( (a, b) => Math.abs(0 - a) - Math.abs(0 - b))[0];
+
+console.log(closest([-3, -2, 6, 8, -1, 1, 12]));
```

Comment:
what if I'll give you numbers instead of array are they work ?

Reply:
+